openapi/anchor-platform/bundled.yml

```diff
@@ -431,14 +461,17 @@ components:
             - withdrawal
         status:
           $ref: '#/components/schemas/StatusSEP24'
+        type:
```

Comment:
Should it not be deprecated for SEP-6? Can't we get it with SEP-12 as a financial field?

Reply:
The business would need the type to know what financial fields to request from the user, right?

Reply:
Yes, but now we have the idea of dynamic KYC. We can request financial field type first, and then request the remaining fields depending on it

Reply:
We discussed this offline. AP does not need to store the `type` of the transaction as wallets make SEP-12 requests with `transaction_id` as well as `type`. This would require a protocol change in potentially SEP-6 and SEP-9 so we will keep passing this field for now and investigate changing this in 4.0.